localenv/mock-account-servicing-entity/app/lib/parse_config.server.ts

```diff
@@ -42,7 +42,9 @@ export interface Account {
 
 interface Fee {
   fixed: number
-  percentage: number
+  basisPoints: number
+  asset: string
+  scale: number
```

Comment:
If this is the interface from the seed file, I'm not sure if we need `asset` & `scale`?

Reply:
Seems like we can also remove the `interface Fees` just above